app/views/reports/index.html.erb

```diff
@@ -8,6 +8,6 @@
   <div class="progress-bar progress-bar-striped"</div>
 </div>
  -->
-<%= render_async patients_report_path(timeframe: 'weekly') %>
-<%= render_async patients_report_path(timeframe: 'monthly') %>
-<%= render_async patients_report_path(timeframe: 'yearly') %>
+<%= render_async patients_report_path(timeframe: 'weekly', nonce: content_security_policy_script_nonce) %>
```

Comment:
ah, I see. Nonce is an options argument of `render_async`, not `patients_report_path`, so pop it outside of the parens and it should blend. e.g. `render_async patients_report_path(...), nonce: ...`